### PR TITLE
🔧 migrate b-pagination

### PR DESCRIPTION
**Pull request by roiLeo**
part of #5852 

Tested on `/bsx/u/bXmx9pNXZw6dYkwaoafJyTrAZVQ1znGq8boJHC2MfQwgGLxPz?page=1`

⚠️ note: I am aware that pagination don't work on "Created" tab even on beta and prod

- fix design issue
- fix random magic button
- pagination vue composition api

Screenshot 2023-06-30 at 17-09-59 NFT Artist Profile on KodaDot KodaDot - NFT Market Explorer

Screenshot 2023-06-30 at 17-14-59 NFT Artist Profile on KodaDot KodaDot - NFT Market Explorer

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
Like @roiLeo pointed, there are some bugs/mobile design flaws that were already occurring on beta, since we are going for a future profile page redesign, best to wait for to avoid unnecessary workforce use. 

IMO, in the future, the way pages are switched, their behavior when switching it and some other minor aspects should be reevaluated to give users a better experience. 

otherwise, the change to Neo Pagination work as intended.

File: components/rmrk/Gallery/Pagination.vue [hunk 1]
```diff
@@ -14,131 +14,77 @@
...
-        if (this.current > 1) {
-          this.current = this.current - 1
-        }
-        break
-      case 'r':
-        this.goToRandomPage()
-        break
-    }
-  }
-
```
Comment (vikiival) [inline components/rmrk/Gallery/Pagination.vue:83]: and this? 🥺

Comment (vikiival) [reply components/rmrk/Gallery/Pagination.vue:83]: Maybe open an issue

**Review by vikiival — APPROVED** [review]
Otherwise OKI

File: components/rmrk/Gallery/Pagination.vue (hunk 1 above)
Comment (roiLeo) [reply components/rmrk/Gallery/Pagination.vue:83]: #6375

**Pull request merged**